### ISSUE TITLE
Helper mowing and baling not working

### ISSUE THREAD

**Issue by jan-klaassen** [Courseplay/courseplay#6289]
With Coursplay enabled in your mods, you are unable to hire a regular helper when you want to mow with a front mower while puling a baler to mow and bale in one go.
No difference between an active of inactive baler.
As soon as you detach the baler, the helper can be hired again.
Removing CP from the mods solves the issue.

Map: Ravenport
Courseplay v6.02.00075
No Mods except CP
Equipment used to reproduce:
Fendt 1050
Pottinger Novicat 301 ED
Kuhn LSB 12090 D

Log: https://gist.github.com/jan-klaassen/d88af79145f44972d39819622224c826

**Comment by Tensuko** [comment]
Well that is strange, but why exactly do you want to use the normale Helper for this ?

**Comment by jan-klaassen** [comment]
I usually like to go for the simplest solution and for this that is a regular Helper.
I discovered this first On the map No man's land where Courseplay has difficulty creating routes from the field outline. So instead of going through the hassle of moving the tractor, finding the right starting point, etc. I like to use the Helper who, in this case, does the job just fine.
For ease of reproduction I tested this on Ravenport with only default equipment.
